artifactory/services/utils/aqlquerybuilder.go

```diff
@@ -11,24 +11,24 @@ import (
 // Returns an AQL body string to search file in Artifactory by pattern, according the the specified arguments requirements.
 func createAqlBodyForSpecWithPattern(params *ArtifactoryCommonParams) (string, error) {
 	searchPattern := prepareSourceSearchPattern(params.Pattern, params.Target, true)
-	pathFilePairs := createRepoPathFileTriples(searchPattern, params.Recursive)
+	repoPathFileTriples := createRepoPathFileTriples(searchPattern, params.Recursive)
 	includeRoot := strings.Count(searchPattern, "/") < 2
-	pathPairsSize := len(pathFilePairs)
+	repoTripleSize := len(repoPathFileTriples)
```

Comment:
I suggest renaming *repoTripleSize* to *tripleSize*, because repo is just one of the three components of a triple.